src/Item/PluginItem/Interpretations/Interpretation.js

```diff
@@ -252,21 +258,24 @@ class Interpretation extends Component {
     }
 }
 
-const mapDispatchToProps = dispatch => {
-    return {
-        likeInterpretation: data => dispatch(tLikeInterpretation(data)),
-        unlikeInterpretation: data => dispatch(tUnlikeInterpretation(data)),
-        addComment: data => dispatch(tAddInterpretationComment(data)),
-        deleteComment: data => dispatch(tDeleteInterpretationComment(data)),
-        deleteInterpretation: data => dispatch(tDeleteInterpretation(data)),
-    };
-};
+const mapStateToProps = state => ({
+    userId: fromUser.sGetUserId(state),
+    isSuperUser: fromUser.sGetIsSuperuser(state),
```

Comment:
Maybe rename this to userIsSuperUser or something that refers to the user, similarly to userIsOwner above